### PR TITLE
common, include, osd: compile with fmt v9

### PR DESCRIPTION
**Pull request by tchaikov**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/osd/scrubber/pg_scrubber.cc [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #include "messages/MOSDScrubReserve.h"
 #include "osd/OSD.h"
 #include "osd/PG.h"
+#include "include/object_fmt.h"
```
Comment (ronen-fr) [inline src/osd/scrubber/pg_scrubber.cc:22]: not needed, as already included thru osd/osd_types_fmt.h

**Review by ronen-fr — CHANGES_REQUESTED** [review]
LGTM (LVGTM...) apart from that extra #include

**Comment by ronen-fr** [comment]
commit message description?

**Comment by tchaikov** [comment]
> commit message description?

@ronen-fr could you please be more specific? i think each commit in this PR has commit message. if any of them missing or just wrong, please let me know.

File: src/osd/scrubber/pg_scrubber.cc (hunk 1 above)
Comment (tchaikov) [reply src/osd/scrubber/pg_scrubber.cc]: sure. dropped.

**Comment by tchaikov** [comment]
@ronen-fr fixed and repushed. could you take another look?

**Comment by ronen-fr** [comment]
Sorry - meant the PR message, not the specific commits.

**Review by ronen-fr — APPROVED** [review]

**Comment by ronen-fr** [comment]
jenkins retest this please

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by cbodley** [comment]
jenkins test api

**Comment by tchaikov** [comment]
jenkins test api

**Pull request merged**